resources/faq.md

```diff
@@ -154,14 +135,10 @@ Good:
 
 See more [information about the primary key](/learn/core_concepts/documents.md#primary-key).
 
-## I have uploaded my documents, but I get no result when I search in my index.
```

Comment:
This FAQ answer is particularly important, since it's a common problem for first-time users of MeiliSearch—whether because their document addition failed, or because it simply hasn't been processed yet. We definitely need some form of this question in the FAQ.